vent/api/menu_helpers.py

```diff
@@ -33,7 +33,7 @@ def cores(self, action, branch="master", version='HEAD'):
             core = self.tools_status(True, branch=branch, version=version)[1]
             if action in ["install", "build"]:
                 tools = []
-                plugins = Plugin(plugins_dir=".internals/plugins/")
+                plugins = Plugin(plugins_dir="plugins/")
```

Comment:
that is the default value for `plugins_dir` so all you need is `Plugin()`, but since that's already an object in this class, you can use `self.plugin` in place of `plugins